src/aequitas/flow/methods/preprocessing/label_flipping.py

```diff
@@ -0,0 +1,241 @@
+from .preprocessing import PreProcessing
+
+from ...utils import create_logger
+from ...utils.imports import import_object
+
+import inspect
+import pandas as pd
+from typing import Optional, Tuple, Literal, Union, Callable
+import numpy as np
+from sklearn.ensemble import BaggingClassifier
+
+METHODS = ["ensemble_margin", "residuals"]
+
```

Comment:
Some things, such as having two lines between a class and what comes before, is done automatically by a tool named `black`. Check for the extension `black Formatter` in VSCode, and change this setting: 
![image](https://github.com/dssg/aequitas/assets/36162088/668f7f55-7d06-4343-b24c-dc61641c761c)